### DISCUSSION TITLE
Outlook Profiles not loading correctly with ghosts?

### DISCUSSION THREAD

**Discussion by 3ch035** [cmu-sei/GHOSTS#478] [Q&A]
Trying to emulate multiple users with a single host with the outlookv2.
With multiple profiles present, I can use a command to update the default profile, and then run outlook which will open with said profile. 

But to do this with ghost it hangs onto the old profile somehow? The script is powershell and it shut downs ghosts and outlook just in case, makes the profile change, starts ghosts back up. Profile doesnt change? Any thoughts on why this is the case?

**Comment by sei-dupdyke** [comment]
Hmm, not sure. We only use 1:1 user:outlook here — anyone else using it differently?
